controllers/add_pods.go

```diff
@@ -97,7 +98,7 @@ func (a AddPods) Reconcile(r *FoundationDBClusterReconciler, context ctx.Context
 				}
 				ip := service.Spec.ClusterIP
 				if ip == "" {
-					log.Info("Service does not have an IP address", "namespace", cluster.Namespace, "cluster", cluster.Name, "podName", pod.Name)
+					logger.Info("Service does not have an IP address", "podName", pod.Name)
```

Comment:
Instead of the pod name we should log here the process group ID to be consistent with the rest of the logging system.